server/services/caseloadService.ts

```diff
@@ -142,6 +147,21 @@ export default class CaseloadService {
       })
   }
 
+  private async mapLicencesAndResponsibleComs(licences: LicenceSummary[]): Promise<LicenceAndResponsibleCom[]> {
+    const comUsernames = licences.map(licence => licence.comUsername)
+    const coms = await this.communityService.getStaffDetailsByUsernameList(comUsernames)
```

Comment:
For each licence, get the responsible COMs in bulk, and match them to the licences based on username

Reply:
Could it be quite a big caseload? Lots of matching to do? Any impact on performance?

Reply:
I don't think so, the COMs are fetched in bulk, and javascript core functions used to map and find, so should be efficiencies built-in

Reply:
Can test it out though

Reply:
Lets do that in DEV or preprod?